lib/galaxy/webapps/galaxy/buildapp.py

```diff
@@ -128,6 +129,9 @@ def paste_app_factory( global_conf, **kwargs ):
 
     register_postfork_function(postfork_setup)
 
+    for th in threading.enumerate():
+        if th.is_alive():
+            log.debug("Prior to webapp return, Galaxy thread %s is alive.", th)
```

Comment:
Maybe a `sys.path` issue? The `uwsgi` importable is part of the uWSGI "interpreter" itself, `uwsgidecorators` is an external module installed into `site-packages`.